### ISSUE TITLE
Scroll.animateScroll.scrollTo multiple components at once

### ISSUE THREAD

**Issue by alexdemartos** [fisshy/react-scroll#253]
I'm trying to animate scroll of two different components at the same time (Scroll.animateScroll.scrollTo).

If there is only one component it works fine. When there's two or more, only one of them is changing the scroll.

**Comment by fisshy** [comment]
Intressting, I'm not sure you can scroll mutiple components, unless you declare a container for each scroll.

**Comment by alexdemartos** [comment]
It is indeed multiple instances of the same component, each of them having a different container for scrolling.

**Comment by alexdemartos** [comment]
Sorry for my inexperience with git and npm. I've updated the react-scroll
package to version 1.6.7 but it is still not fixing it for me. Does this
version include the last fixes on this matter?

2017-11-28 13:43 GMT+01:00 Nikolai Kim <notifications@github.com>:

**Comment by yaruson** [comment]
No, the PR is not merged to master branch yet. You may still try it like this: in your `package.json` replace version number with

`"react-scroll": "github:yaruson/react-scroll#multiple-scrolls-build"`

**Comment by alexdemartos** [comment]
Working like a charm! Thank you very much, this saved my day =) It should be pushed to the main branch

**Comment by yaruson** [comment]
@alexdemartos now the changes are in v1.7.0, so please make appropriate changes into your dependencies and update the issue status once it is resolved 😸
